sdk/cognitivelanguage/azure-ai-language-questionanswering/samples/authoring/async_samples/sample_export_import_project_async.py

```diff
@@ -16,66 +16,55 @@
     Set the environment variables with your own values before running the sample:
     1) AZURE_QUESTIONANSWERING_ENDPOINT - the endpoint to your QuestionAnswering resource.
     2) AZURE_QUESTIONANSWERING_KEY - your QuestionAnswering API key.
+    3) AZURE_QUESTIONANSWERING_PROJECT - your existing Question Answering project to export/import.
 """
 
 import asyncio
 
+
 async def sample_export_import_project_async():
     # [START export_import_project]
     import os
+    import io
+    import zipfile
+    from azure.core.rest import HttpRequest
     from azure.core.credentials import AzureKeyCredential
     from azure.ai.language.questionanswering.authoring.aio import AuthoringClient
 
     # get service secrets
     endpoint = os.environ["AZURE_QUESTIONANSWERING_ENDPOINT"]
     key = os.environ["AZURE_QUESTIONANSWERING_KEY"]
+    project_name = os.environ["AZURE_QUESTIONANSWERING_PROJECT"]
 
     # create client
     client = AuthoringClient(endpoint, AzureKeyCredential(key))
     async with client:
 
-        # create project
-        project_name = "IssacNewton"
-        await client.create_project(
-            project_name=project_name,
-            options={
-                "description": "biography of Sir Issac Newton",
-                "language": "en",
-                "multilingualResource": True,
-                "settings": {
-                    "defaultAnswer": "no answer"
-                }
-            })
-
```

Comment:
I take it these samples don't actually run as part of tests like ours typically do? If not, great; otherwise, what sets the env var to an existing project so you can exercise exporting a project?

Reply:
The env var is set on L37 for the existing project.

Reply:
On second thought, you're totally right that this isn't going to work with parallel live tests. I'll skip these for now.